packages/common/src/types/TextEditor.ts

```diff
@@ -149,9 +149,8 @@ export interface EditableTextEditor extends TextEditor {
 
   /**
    * Paste clipboard content
-   * @param ranges A list of {@link Range ranges}
    */
-  clipboardPaste(ranges?: Range[]): Promise<void>;
```

Comment:
This was always a lie. Under no circumstances did we call this function with a range.